packages/backend/src/modules/match/report-match-win.mutation.ts

```diff
@@ -26,17 +26,15 @@ export default class ReportMatchWinMutationResolver {
 			throw new Error('ReportMatchWin: Match does not exist');
 		}
 
+		if (match.finalized) throw new Error('Match already finalized ');
+
 		const teamRepository = getRepository(DBTeam);
 		const team = await teamRepository.findOne(teamId);
 
 		if (!team) {
 			throw new Error('ReportMatchWin: Team does not exist');
 		}
 
-		if (match.winner) {
```

Comment:
We deleted it because we need to set the value again. If we still had this condition we would not be able to set the value to something else if the admin wants to change the winning team